### PR TITLE
Update bootstrap: Reports

### PR DESCRIPTION
**Pull request by maiwald**

### CODE REVIEW

File: app/packs/src/apps/mydb/elements/details/reports/OrdersDnD.js [hunk 1]
```diff
@@ -40,35 +39,18 @@ const orderDropCollect = (connect, monitor) => (
...
+}) => {
   const style = {};
-  if (canDrop) {
-    style.borderStyle = 'dashed';
-    style.borderWidth = 2;
-  }
   if (isOver) {
-    style.borderColor = '#337ab7';
+    style.borderStyle = 'dashed';
+    style.borderWidth = '2px';
```
Comment (beque) [inline app/packs/src/apps/mydb/elements/details/reports/OrdersDnD.js:53]: We use a border width of 3px at other drag'n'drop zones

File: app/packs/src/apps/mydb/elements/details/reports/SectionSiSynthesis.js [hunk 2]
```diff
@@ -30,12 +29,10 @@ const Title = ({ el, counter, molSerials }) => {
       ? [...title, <span key={key}>{smn} (<b>{us}</b>)</span>, comma]
       : [...title, <span key={key}>&quot;<b>NAME</b>&quot; (<b>{us}</b>)</span>, comma];
   });
-  title = _.flatten(title).slice(0, -1);
+  title = title.slice(0, -1);
```
Comment (beque) [inline app/packs/src/apps/mydb/elements/details/reports/SectionSiSynthesis.js:32]: Why did you remove `_.flatten`?
I don't know which kind of title could occur. But if you have an array in an array the result could be different

File: app/packs/src/apps/mydb/elements/details/reports/SectionSiSynthesisStdRxn.js [hunk 3]
```diff
@@ -30,12 +29,10 @@ const Title = ({ el, counter, molSerials }) => {
       ? [...title, <span key={key}>{smn} (<b>{us}</b>)</span>, comma]
       : [...title, <span key={key}>&quot;<b>NAME</b>&quot; (<b>{us}</b>)</span>, comma];
   });
-  title = _.flatten(title).slice(0, -1);
+  title = title.slice(0, -1);
```
Comment (beque) [inline app/packs/src/apps/mydb/elements/details/reports/SectionSiSynthesisStdRxn.js:32]: Here the same

**Review by beque — COMMENTED** [review]
Just some minor fixes

File: app/packs/src/apps/mydb/elements/details/reports/SectionSiSynthesis.js (hunk 2 above)
Comment (maiwald) [reply app/packs/src/apps/mydb/elements/details/reports/SectionSiSynthesis.js:32]: We create the `title` in line 22 and only ever "append" (i.e. `[…title, ...]`) two `<span>` Tags to it. So there are no nested arrays.
